.github/workflows/e2e-tests.yml

```diff
@@ -26,10 +26,3 @@ jobs:
           AUTH0_DOMAIN: ${{ secrets.AUTH0_DOMAIN }}
           AUTH0_CLIENT_ID: ${{ secrets.AUTH0_CLIENT_ID }}
           AUTH0_CLIENT_SECRET: ${{ secrets.AUTH0_CLIENT_SECRET }}
-
-      - name: Update codecov report
-        uses: codecov/codecov-action@v3
-        with:
-          files: ./coverage.out
-          fail_ci_if_error: false
-          verbose: true
```

Comment:
Codecov report intentionally removed?

Reply:
Yes due to several reasons:
- We have to run a go test per package manually to ensure tests run one at a time
- Considering the high chance of intermittent errors due to the nature of e2e tests there's no point in uploading the coverage for the e2e tests

Codecov report will only get pushed on the full test suite run with http recordings.